capsules/src/process_console.rs

```diff
@@ -170,6 +174,68 @@ impl Default for WriterState {
     }
 }
 
+#[derive(Copy, Clone)]
+pub struct Command<const LEN: usize> {
+    buf: [u8; LEN],
```

Comment:
```suggestion
pub struct Command {
    buf: [u8; COMMAND_BUF_LEN],
```

To make sure that the whole command fits, use the `COMMAND_BUF_LEN` used for the reading buffer.

Reply:
When creating the command in components::process_console I initialize the Command structure with Command<{capsules::process_console::COMMAND_BUF_LEN}> which is the same, should I change it?

Reply:
I suggest deleting the generic and always use that value. Right now you do you that value, but someone else might use a different one.

Reply:
Done